### PR TITLE
Renamed File Explorer to File Manager

### PR DESCRIPTION
**Pull request by pixl-8**
With the name 'File Explorer', it seems like SkiffOS is a clone of windows or a wannabe Windows. This is not how I believe it should be.

### CODE REVIEW

File: applications/file-manager/.build.mk [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
+APPS += FILE_MANAGER
 
-FILE_EXPLORER_NAME = file-explorer
-FILE_EXPLORER_LIBS = widget graphic json
-FILE_EXPLORER_ICONS = \
-	folder \
+FILE_MANAGER_NAME = file-manager
+FILE_MANAGER_LIBS = widget graphic json
+FILE_MANAGER_ICONS = \
+	folder
```
Comment (NicolasVanBossuyt) [inline applications/file-manager/.build.mk:6]: You are missing \ at the end of the line

**Review by NicolasVanBossuyt — CHANGES_REQUESTED** [review]

File: applications/file-manager/.build.mk (hunk 1 above)
Comment (pixl-8) [reply applications/file-manager/.build.mk]: All fixed 👍

**Comment by NicolasVanBossuyt** [comment]
LGTM :+1:

**Review by pixl-8 — COMMENTED** [review]
All good

**Pull request merged**